src/typescript/13-utility-types/utility-types.md

```diff
@@ -105,9 +145,9 @@ dino.age += 1;
 // Oops! TypeScript error
 ```
 
```

Comment:
Every Objective should end with the exercise, then a new Objective should start.